### PR TITLE
Fix missing type in _add_attribute_to_torchscript_node for Deberta models

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: onnxscript/function_libs/torch_lib/graph_building/_graph_building_torch.py [hunk 1]
```diff
@@ -459,8 +459,19 @@ def _add_attribute_to_torchscript_node(
...
-        raise TypeError(f"Unsupported sequence type '{type(value)}' for attribute '{key}'")
-    raise TypeError(f"Unsupported attribute type '{type(value)}' for attribute '{key}'")
+        raise TypeError(
+            f"Unsupported sequence type '{type(value)}' for attribute '{key}' in "
+            f"node={node!r}, value is {value!r}"
+        )
+    if "TensorProtoDataType" in str(type(value)):
+        # torch._C._onnx.TensorProtoDataType
+        itype = int(value)
+        return node.i_(key, itype)
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_lib/graph_building/_graph_building_torch.py:469]:
```suggestion
        return node.i_(key, int(value))
```

File: onnxscript/function_libs/torch_lib/graph_building/_graph_building_torch.py [hunk 2]
```diff
@@ -459,8 +459,19 @@ def _add_attribute_to_torchscript_node(
             return node.fs_(key, list(value))  # type: ignore[arg-type]
         if isinstance(value[0], int):
             return node.is_(key, list(value))  # type: ignore[attr-defined]
-        raise TypeError(f"Unsupported sequence type '{type(value)}' for attribute '{key}'")
-    raise TypeError(f"Unsupported attribute type '{type(value)}' for attribute '{key}'")
+        raise TypeError(
+            f"Unsupported sequence type '{type(value)}' for attribute '{key}' in "
+            f"node={node!r}, value is {value!r}"
+        )
+    if "TensorProtoDataType" in str(type(value)):
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_lib/graph_building/_graph_building_torch.py:466]: I still don't understand why we would see `torch._C._onnx.TensorProtoDataType` here. Do you know what the caller is?

File: onnxscript/function_libs/torch_lib/graph_building/_graph_building_torch.py (hunk 1 above)
Comment (xadupre) [reply onnxscript/function_libs/torch_lib/graph_building/_graph_building_torch.py]: done

File: onnxscript/function_libs/torch_lib/graph_building/_graph_building_torch.py (hunk 2 above)
Comment (xadupre) [reply onnxscript/function_libs/torch_lib/graph_building/_graph_building_torch.py:466]: I did not find any issue with the caller. I think this issue can be fixed by inserting an exception when the container for the attribute is created and then you would know why this type is here. I did this fix to validate the fix works. The type is an int enum and I assume in many places it is considered as such except here. The best fix would be to catch this issue when this value enters the IR but you know better than me all the places where it happens.

**Review by justinchuby — APPROVED** [review]

**Pull request merged**